.sh.d/01_nvda2svn.sh

```diff
@@ -9,7 +9,7 @@ makePot() {
         --foreign-user --add-comments=Translators: --keyword=pgettext:1c,2 \
         --from-code utf-8 \
         --language=python \
-        *.py *.pyw */*.py */*/*.py
+        *.py *.pyw */*.py */*/*.py */*/*/*.py
```

Comment:
perhaps we should just use `**/*.py`?

Is that supported here?

Reply:
I do not know if **.py is supported here so I have taken the simplest (but lesss robust) approach.

Of course, if one more level is added, the current approach will fail for the same reason and a new level will have to be added.

I have no objection to use **.py if it works. I have no way to test it on NVAccess server, so may you test yourself please?

Reply:
Unfortunately it doesn't work

Reply:
```suggestion
        # Unfortunately **/*.py doesn't work here
        *.py *.pyw */*.py */*/*.py */*/*/*.py
```